plugins/backend/BackendBlockly.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3

```

Comment:
my class to integrate and render the Blockly Logics Editor functionality inside backend plugin.